libs/dh/esett/feature-outgoing-messages/src/lib/drawer/dh-outgoing-message-drawer.component.html

```diff
@@ -59,15 +59,17 @@ <h2 class="message-heading">
       </watt-description-list>
     </watt-drawer-heading>
 
-    <watt-drawer-content *ngIf="drawer.isOpen">
-      <watt-tabs>
-        <watt-tab *ngIf="dispatchDocument" [label]="t('tabs.message')">
-          <watt-code [code]="dispatchDocument | push" />
-        </watt-tab>
-        <watt-tab *ngIf="responseDocument" [label]="t('tabs.receipt')">
-          <watt-code [code]="responseDocument | push" />
-        </watt-tab>
-      </watt-tabs>
-    </watt-drawer-content>
+    @if (drawer.isOpen) {
+      <watt-drawer-content>
+        <watt-tabs>
+          <watt-tab [label]="t('tabs.message')">
```

Comment:
question: the reason for having `ngIf` before was to avoid rendering a potentially big XML/JSON message in a tab that is not active. Not both messages will always be rendered. With a WebWorker this will hopefully not be a performance issue, but we should be aware of that. Especially because the messages here can be quite big.